lib/prepArticles.js

```diff
@@ -39,19 +34,18 @@ function prepAll(store, category) {
 	
 					scrape(article.url , parsedResult => {
 						if (!parsedResult) {
+							// TODO: These articles are still being sent to frontend, which they shouldn't be
 							console.error(`Error scraping article: ${article.url}`)
 							completedArticles++
 						}
 						else {
-							completedArticles++
-
-							console.log("Done Scraping: " + completedArticles + " / " + articleList.length + "\t(" + category + ")")
-	
 							newArticle.text = parsedResult.content
 							newArticle.timeToRead = Math.ceil(parsedResult.word_count / AVG_WORDS_PER_MINUTE)
 							newArticle.sentiment = sentiment(parsedResult.content)
 							result.push(newArticle)
+							completedArticles++
 						}
+						console.log("Done Scraping: " + completedArticles + " / " + articleList.length + "\t(" + category + ")")
```

Comment:
This should still be inside, it indicates a successful scrape? Or otherwise maybe change it to be more clear that it isn't "done scraping" but it has passed this one idk

Reply:
Changed 👍